### DISCUSSION TITLE
Function drawWedgeLine doesn't support squared ends

### DISCUSSION THREAD

**Discussion by janci78** [Bodmer/TFT_eSPI#3019] [General]
Hi there, I've realized that **drawWedgeLine** function supports just rounded ends unlike the **drawSmoothArc** function which enables rounded ends with a special "roundEnds" parameter. 

Question: Is there a way how to get squared ends with **drawWedgeLine**? Any handy workaround or suggestion if the function doesn't support it?
Any help with it would be much appreciated.
Cheers, 
Jan

**Comment by Bodmer** [comment]
No, that is not a capability yet. I created that function for dial pointers.

Eventually a general purpose smotth polygon function will be added but I do not have time to do that at the moment due to other priorities.

**Comment by janci78** [comment]
thanks for your fast response. Understood ... never mind, will write up something what does the trick. Cheers.
